source/jormungandr/tests/tests_mechanism.py

```diff
@@ -236,8 +236,14 @@ def query_no_assert(self, url, display=False):
     def check_journeys_links(self, response, query_dict):
         journeys_links = get_links_dict(response)
         clockwise = query_dict.get('datetime_represents', 'departure') == "departure"
+        has_pt = any(s for j in response['journeys'] for s in j['sections'] if s['type'] == 'public_transport')
```

Comment:
I think what you want is this: `has_pt = any(s['type'] == 'public_transport' for j in response['journeys'] for s in j['sections'])`